infra/azure/main.tf

```diff
@@ -442,13 +479,53 @@ module "ci_sp" {
 
 resource "azuread_application" "test" {
   display_name = "${data.azurerm_resource_group.rg.name}-test"
+
+  required_resource_access {
+    resource_app_id = "00000003-0000-0000-c000-000000000000"
+
+    resource_access {
+      # Application.ReadWrite.All
+      id   = "1bfefb4e-e0b5-418b-a88f-73c46d2cc8e9"
+      type = "Role"
+    }
```

Comment:
Huh. Can you give some context on why we write a resource_access here? I would have expected an azurerm_role_assignment. Also, what does "Application.ReadWrite.All" mean? Is that full access to Azure AD Applications?